pyocd/target/builtin/target_MAX32660.py

```diff
@@ -0,0 +1,91 @@
+# pyOCD debugger
+# Copyright (c) 2017-2021 Arm Limited
```

Comment:
This copyright should be changed to your own (or employer).

Reply:
Done.